### DISCUSSION TITLE
Fatal Error - Invalid class

### DISCUSSION THREAD

**Discussion by ewalgran** [SafeExamBrowser/seb-win-refactoring#657] [Q&A]
Student installed SEB 3.5.0.544 and when launching receives a Fatal Error - invalid class message. Looking for guidance on how to resolve this issue. 

![image](https://github.com/SafeExamBrowser/seb-win-refactoring/assets/128709895/cf80eef5-f122-4204-8f64-5499f859e032)

**Comment by dbuechel** [comment]
There are already quite a few resources regarding this issue here on GitHub, please search for them and try the steps outlined there. Up to now, it most of the time turned out to be a (configuration) issue on a particular machine resp. system.
